src/Month.java

```diff
@@ -0,0 +1,30 @@
+public enum Month {
+    JANUARY(30),
+    FEBRUARY(28),
+    MARCH(31),
+    APRIL(30),
+    MAY(31),
+    JUNE(30),
+    JULY(31),
+    AUGUST(31),
+    SEPTEMBER(30),
+    OCTOBER(31),
+    NOVEMBER(30),
+    DECEMBER(31);
+
+
+    private final int days;
+
+    Month(int days) {
+        this.days = days;
+    }
+
+    public int getSpot(Month m) {
+        return m.ordinal();
+    }
```

Comment:
```suggestion
    public int getSpot() {
        return ordinal();
    }
```
please don't resolve comments without changing anything, if you don't understand a comment ask

Reply:
probably forgot to commit push so it didnt save it, sorry, my bad